src/proxy-ui/Gemfile.lock

```diff
@@ -91,7 +89,7 @@ GEM
     treetop (1.4.15)
       polyglot
       polyglot (>= 0.3.1)
-    tzinfo (0.3.54)
+    tzinfo (0.3.52)
```

Comment:
This file contains many downgrades for Ruby gems

Reply:
Fixed https://github.com/nordic-institute/X-Road/commit/834c83c8a9159eea3d9a6577bf5dfa2c580859c3

Background: https://niis.slack.com/archives/GGBHVCYSY/p1573477809003000?thread_ts=1573474620.001500&cid=GGBHVCYSY

Reply:
Running jenkins build https://jenkins.niis.org/view/build/job/xroad-build-packages-pipeline/1122/